src/ws/session.c

```diff
@@ -78,17 +82,19 @@ read_auth_until_eof (size_t *out_len)
       if (alloc <= len)
         {
           alloc += 1024;
+          if (alloc > 64 * 1024)
```

Comment:
should we have a define at the top of the file / somewhere central for limits like these?
at least document the hard limit for the function